### ISSUE TITLE
index document invalid for static document mapping

### ISSUE THREAD

**Issue by heidawei** [blevesearch/bleve#946]
in func (dm *DocumentMapping) documentMappingForPath(path string) *DocumentMapping 
// no subDocMapping matches this pathElement
		// only if this is the last element check for field name
		if i == len(pathElements)-1 {
			for _, field := range current.Fields {
				fmt.Println("field name ", field.Name, pathElement)
				if field.Name == pathElement {
					break OUTER
				}
			}
		}
for static document mapping,  we need break OUTER not current for{}, it will return nil

**Comment by mschoch** [comment]
Yes, your suggestion looks correct.  However, I'd like to add a unit test to verify it works the way we look, and make sure the other tests continue to pass.  I'll try to get to this today.
